ClinGen/clincoded
# Connects to #682. Connects to #684. CAR parsing/handling

![image](https://cloud.githubusercontent.com/assets/4326866/15694253/f22bdb06-2750-11e6-93da-8c3980bbe9bf.png)

Testing:

1. Go to select-variant page
2. Select CA ID option
  * Confirm that links out to CAR are all correct both in instructions and modal
3. Test modal w/ CA IDs [CA004123](http://reg.genome.network/allele/CA004123) and [CA009941](http://reg.genome.network/allele/CA009941) - should actually return and save ClinVar data along with CA ID
4. Test modal w/ CA IDs [CA033442](http://reg.genome.network/allele/CA033442), [CA001400](http://reg.genome.network/allele/CA001400) - should return and save CAR data as there is no associated ClinVar data, along with reference genome data
5. Test modal w/ CA ID [CA091413](http://reg.genome.network/allele/CA091413) - should return and save CAR data a there is no associated ClinVar data, and no reference genome data
6. Test modal w/ CA ID [CA999999](http://reg.genome.network/allele/CA999999) - should return a CA ID not found error (connects to #684 specifically)
